corehq/pillows/mappings/case_search_mapping.json

```diff
@@ -1,6 +1,6 @@
 {
     "date_detection": false,
-    "date_formats": "__DATE_FORMATS_ARR__",
+    "date_formats": $DATE_FORMATS_ARR,
```

Comment:
Ah, tried implementing this... this isn't valid JSON (as you mentioned), so I'm going to leave it as is for now since it kind of defeats the purpose otherwise. :disappointed: